demisto_sdk/commands/common/hook_validations/structure.py

```diff
@@ -8,7 +8,7 @@
 from typing import Optional
 
 import yaml
-from pykwalify.core import Core
+from pykwalify.core import Core, logging
```

Comment:
this might cause some issues in the future if we would like to use `logging`
lets just `import logging` (not from `pykwalify`) and capture the required logger using `logging.getLogger`

Reply:
changed it